### DISCUSSION TITLE
How should I do to access other folders by RNFS.exists()?

### DISCUSSION THREAD

**Discussion by gangli66** [microsoft/react-native-windows#10522] [Q&A] [answered]
I use react-native-webview in my react-native-windows project.
So when the webview is loaded, it downloads static contents from web server to local cache folders.
But the webview cache folder is different from appx cache folder.
When I use react-native-fs function RNFS.exists() to judge if the web cache folder
(C:\Users\xxx\AppData\Local\Package\myproject\AC\) exists,
it catches an error that access denied.
I found a solution from microsoft website:
Adding 
1) xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities",
2) IgnorableNamespaces="uap mp iot rescap"
3) <rescap:Capability Name="broadFileSystemAccess" />
to Package.appxmanifest.
But now the site http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities return error like
The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.
How should I do to access other folders by RNFS.exists()?

**Comment by chrisglein** [comment, marked as answer]
The capability of "broadFileSystemAccess" is a restricted capability that I don't believe most apps can attain. UWP apps are expected to work in a tighter container than that. So generally, you won't have access to the entire file system.

What is your scenario?

**Reply by gangli66** [reply]
> The capability of "broadFileSystemAccess" is a restricted capability that I don't believe most apps can attain. UWP apps are expected to work in a tighter container than that. So generally, you won't have access to the entire file system.
> 
> What is your scenario?

Thanks for your reply!
I used WebView in my react-native-windows project and it will download static files from web to cache. I have found the cache folder is C:/Users/myaccount/AppData/Local/packages/myproject/AC/INetCache. I need to delete those cache when a user chooses "clear cache = on". But I cound't access .../AC. After I choose File System Access ON, I could access .../AC, but I still couldn't access .../AC/INetCache by RNFS.exists() because it is a hidden folder. I used dos command to make it showed, but when the webview was opened, it became hidden folder again. 
The error that RNFS.exists() returning was "Access Denied".
